BGAnimations/ScreenSelectMusic overlay/SortMenu/SortMenu_InputHandler.lua

```diff
@@ -224,7 +224,12 @@ local input = function(event)
 						-- loaded consistently.
 						-- TODO(teejusb): Fix this in the engine.
 						-- MESSAGEMAN:Broadcast("Sort", { order = "Preferred" })
-						
+							
+						-- If the MenuTimer is in effect, make sure to grab its current
+						-- value before reloading the screen.
+						if PREFSMAN:GetPreference("MenuTimer") then
+							overlay:playcommand("ShowPressStartForOptions")
```

Comment:
Hmm the comment doesnt make it obvious why `playcommand("ShowPressStartForOptions")` preserves the timer value.

Reply:
Yeah I think that's better. I noticed that this code is actually copied from above so maybe change it there too.

Reply:
Also I changed the base to `itgmania-beta` and it needs to be cleaned up again >_>

Reply:
Alrighty, should be cleaned up and ready